docs/install/README.md

```diff
@@ -2,3 +2,5 @@
 To install Knative components on your Kubernetes cluster, follow our [installation guide](./knative-with-any-k8s.md).
 
 To install the Knative CLI, follow our [installation guide](./install-kn.md).
+
+To install Knative components with Knative Operators, follow our [installation guide with Knative Operators](./knative-with-operators.md).
```

Comment:
I think this might be clearer if you make this an alternative on line 2, rather than a separate item. Right now, this reads:

1. Follow these instructions to install Knative
2. Then install the client side tools
3. Then install the operator.

I think you want:

1. Install Knative manually, or use the operator
2. Then install the CLI tools.

Reply:
done.